ui/app/pages/swaps/fee-card/fee-card.js

```diff
@@ -22,6 +22,30 @@ export default function FeeCard ({
             <div className="fee-card__row-header-text--bold">
               {t('swapEstimatedNetworkFee')}
             </div>
+            <div className="fee-card__row-label fee-card__info-tooltip-container">
```

Comment:
`Tooltip` exposes two props that would make two of these divs obsolete if we add them to `InfoTooltip`. `containerClassName` could be set to `fee-card__info-tooltip-content-container` and `wrapperClassName` could be set to `fee-card__row-label fee-card__info-tooltip-container` and then two extra divs are no longer necessary.